### DISCUSSION TITLE
Android ABI - x86/x64 disable inclusion in AAB archive

### DISCUSSION THREAD

**Discussion by teotigraphix** [airsdk/Adobe-Runtime-Support#2509] [Q&A] [answered]
I have native extensions that are compiled with NDK 32/64 ARM support but my compiles are not working atm moment with x86/64, I think NDK versions need to be the same or something, first time I have looked into this ABI.

So is there any way to exclude those ABIs being added to the AAB, currently Play dashboard shows I have 4 ABIs which if they dished out x86/64 my app would crash because it's missing my native libraries. I know I can see them opening up the aab archive.

I guess we need a way to toggle off ABI support?

**Comment by ajwfrost** [comment, marked as answer]
Yes, it should be possible to do this, see https://airsdk.dev/docs/building/application-descriptor-files/elements/android#buildarchitectures

So basically in your app descriptor file, if you have your `<android>` section, you can add:
```
<buildArchitectures>armv7,armv8</buildArchitectures>
```

thanks

**Reply by teotigraphix** [reply]
Thanks, looks like I need to read over the list again. :) Missed that tag.

Edit: Was thinking compiler "flags" this whole time as well.

**Reply by teotigraphix** [reply]
Check; I just compiled an .aab and it works as expected, thanks @ajwfrost !.

**Reply by ajwfrost** [reply]
Phew :-) Always good to have real-world confirmation!
